lib/retry.ts

```diff
@@ -62,6 +63,11 @@ export interface RetryConfig<T> {
    * next attempt. Default: 15.
    */
   delayInSeconds?: number;
+  /**
+   * @property {string} connectionHost The host "<yournamespace>.servicebus.windows.net".
+   * Used to check network connectivity.
+   */
+  connectionHost?: string;
```

Comment:
Will this be populated by the user or is this something we will populate based on the endpoint?

Reply:
This will be populated by the consumer of this package which in our case will be the Event Hubs and Service Bus SDK